Tests/runTestSimulation.m

```diff
@@ -44,14 +42,11 @@
     if strcmpi(varargin{i},'run_as_batch_job')
         run_as_batch_job = varargin{i+1};
     end
-    if strcmpi(varargin{i},'PathCpp2Dll_Exe')
-        PathCpp2Dll_Exe = varargin{i+1};
-    end
     if strcmpi(varargin{i},'CasADi_path')
         CasADi_path = varargin{i+1};
     end
     if strcmpi(varargin{i},'Cpp2Dll_compiler')
```

Comment:
should line 48 be if strcmpi(varargin{i},'OpenSimAD_compiler')?

Reply:
good catch